src/app/flows/fines/fines-acc/fines-acc-defendant-details/fines-acc-party-details/fines-acc-party-details.component.html

```diff
@@ -76,9 +76,11 @@
         </div>
 
         @if (party.vehicle_details; as vehicle) {
-          <ng-container
-            *ngTemplateOutlet="vehicleDetailsTemplate; context: { vehicle: vehicle, summaryListId: summaryListId }"
-          ></ng-container>
+          @if (party.is_debtor) {
+            <ng-container
+              *ngTemplateOutlet="vehicleDetailsTemplate; context: { vehicle: vehicle, summaryListId: summaryListId }"
+            ></ng-container>
+          }
         }
```

Comment:
I think the if statement needs to be is_debtor first then if vehicle_details thinking about it logically

This way it only shows if the party is the debtor whereas if it was vehicle_details first you complete that if first then the is debtor when we can't display it anyway. 

Or another way of doing it is `@if (party.is_debtor && party.vehicle_details; as vehicle)` <- might be the better option

```suggestion
       @if (party.is_debtor) {
            @if (party.vehicle_details; as vehicle) {
            <ng-container
              *ngTemplateOutlet="vehicleDetailsTemplate; context: { vehicle: vehicle, summaryListId: summaryListId }"
            ></ng-container>
          }
}
```

Reply:
Ahh yep your right. thats the best way to do it. Thanks mate.